charts/keycloak/values.yaml

```diff
@@ -108,6 +108,7 @@ keycloak:
     -XX:MaxRAMPercentage=50.0
 
   ## Allows the specification of additional environment variables for Keycloak
+  ## Enable KEYCLOAK_HTTP_PORT, KEYCLOAK_HTTPS_PORT and KEYCLOAK_HOSTNAME if you are using ingress: true
```

Comment:
```suggestion
  ## Enable PROXY_ADDRESS_FORWARDING if you are using ingress: true
```

Reply:
I will test again on my dev cluster to see, thanks for the input!

Reply:
So, it deploys right, and you can access the welcome page at mydomain.com/auth/ but the links to the realm management don't work because they have ":8443" at the end of them (even removing the ports and trying to navigate directly doesn't work), so you actually need all 4 env variables set to get ingress to function properly.

- `KEYCLOAK_HTTP_PORT`
- `KEYCLOAK_HTTPS_PORT`
- `KEYCLOAK_HOSTNAME`
- `PROXY_ADDRESS_FORWARDING`